cmd/antrea-agent/agent.go

```diff
@@ -136,6 +136,8 @@ func run(o *Options) error {
 	enableBridgingMode := enableAntreaIPAM && o.config.EnableBridgingMode
 	enableNodePortLocal := features.DefaultFeatureGate.Enabled(features.NodePortLocal) && o.config.NodePortLocal.Enable
 	enableMulticluster := features.DefaultFeatureGate.Enabled(features.Multicluster) && o.config.Multicluster.Enable
+	// L7NetworkPolicy is not supported on Windows yet.
```

Comment:
you should probably remove the comment as well

Reply:
removed, thanks